src/hardware/adc.rs

```diff
@@ -29,8 +26,14 @@ impl AdcCode {
     const R_REF: f32 = 2.0 * 5000.0; // Ratiometric resistor setup. 5.0K high and low side.
     const ZERO_C: f32 = 273.15; // 0°C in °K
     const B: f32 = 3988.0; // NTC beta value. TODO: This should probaply be changeable.
-    const T_N: f32 = 25.0; // Reference Temperature for B-parameter equation.
+    const T_N: f32 = 25.0 + AdcCode::ZERO_C; // Reference Temperature for B-parameter equation.
     const R_N: f32 = 10000.0; // TEC resistance at T_N.
+
+    // ADC relative full scale per LSB
+    // Inverted equation from datasheet p. 40 with V_Ref normalized to 1 as this cancels out in resistance.
+    const FS_PER_LSB: f32 = 0x400000 as f32 / (2.0 * (1 << 23) as f32 * AdcCode::GAIN * 0.75);
```

Comment:
Would you say it is better to move calculations at compile time to constants in general? I think it makes no difference from a code optimization standpoint, right?

Reply:
No difference for code speed. It's just readability and somewhat subjective. To me "full scale units per LSB" is unambiguous, understandable, and well-defined.